Colours.txt

```diff
@@ -99,6 +99,14 @@
 #define smrTETaxiway 7106162
 #define smrTFGrass 4487739
 #define smrTFTaxiway 3498623
+#define smrVAApron 11908532
```

Comment:
To confirm, are any of these colours defined under a different name? If so, can convert them to generic smr colours rather than having the two-letter ICAO designator thing

Reply:
Ah I see, you're defining them as VA elsewhere too - not to worry, think this works

Reply:
Can change the name if needs be - I intend to use the colours for more Mil SMRs in future
Maybe smrMIL_______?

Reply:
when adding the Brize and Coningsby SMRs me and Scott have used smrmil____ as you said above and plan on using them for all the mil airfields we already have so think it would be good to get all the mil airfields using the same set of colours

Reply:
Ok I'll change it on my end, I'm planning on doing one for Culdrose unless you guys have one already?

Reply:
sounds good.